### PR TITLE
Add blank graph component

### PR DESCRIPTION
**Pull request by jnmarcus**
screen shot 2016-02-17 at 7 38 42 pm
screen shot 2016-02-17 at 7 39 43 pm

### CODE REVIEW

File: app/components/common/blankGraph/blankGraphDirective.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      //controller: 'blankGraphController'
+      controller: function() {}
+    };
+    return directive;
+
+    function link(scope, element, attrs, vm, transclude) {
+      element.find('.cfa-blankGraph-content').append(transclude());
+    }
+  };
+})();
```
Comment (adborden) [inline app/components/common/blankGraph/blankGraphDirective.js]: No need for the IIFE closure, browserify handles it for you.

File: app/components/common/pageHeader/pageHeader.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+(function() {
+  'use strict';
+
+  //var pageHeaderModule = require('./pageHeader');
+  var pageHeaderDirective = require('./pageHeaderDirective');
+
+  module.exports = angular.module('pageHeaderModule', [])
+    .directive('pageHeader', pageHeaderDirective);
+
+})();
```
Comment (adborden) [inline app/components/common/pageHeader/pageHeader.js]: No need for the IIFE closure, browserify handles it for you.

File: app/components/common/blankGraph/blankGraph.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<article class="cfa-contentBlock--singleItem">
+	<h3 class="cfa-blankGraph-title">{{vm.graphTitle}}</h3>
+	<div class="cfa-blankGraph-content">
+		<div ng-transclude></div>
+	</div>
+</article>
```
Comment (adborden) [inline app/components/common/blankGraph/blankGraph.html]: Why the `article` tag?

File: app/components/common/blankGraph/blankGraph.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+(function() {
+  'use strict';
+
+  //var blankGraphModule = require('./blankGraph');
+  var blankGraphDirective = require('./blankGraphDirective');
+
+  module.exports = angular.module('blankGraphModule', [])
+    .directive('blankGraph', blankGraphDirective);
+
+})();
```
Comment (adborden) [inline app/components/common/blankGraph/blankGraph.js]: No need for the IIFE closure, browserify handles it for you.

**Comment by adborden** [comment]
Thanks Jamie, this looks good. Just curious what we're going to be using the blank-graph component for? It doesn't look much like a graph in numerical/chart sense.

File: app/components/common/blankGraph/blankGraph.html (hunk 3 above)
Comment (jnmarcus) [inline app/components/common/blankGraph/blankGraph.html]: bc it's a self-containing element with its own header. I've been going back and forth on it, but my thinking for leaving it was bc _technically_ I believe it's the right html structure, which will help with indexing when we start to worry about SEO...thoughts?

**Comment by jnmarcus** [comment]
@adborden this component is just meant to be a blank slate for developers and/or campaign finance people that just want to do d3 things on the site and have it look right, without having to worry about any of the overhead.  This way they can just plop a blank-graph-component in anywhere, add a title for the chart/graph/whatever, and get started on whatever it is that they're actually trying to do.  

I had asked @bcipolli if this was something he thought could be useful sooner rather than later, and he thought so, so I added it in

**Comment by jnmarcus** [comment]
@adborden we seem to have a difference of opinion on whether or not to use the IIFE.  I haven't read anything that explicitly states that Browserify handles the IIFE for you, but even if it does, I still just find it kind of limiting.  Using IIFEs are an extremely common best practice, and if we ever decide that we want to switch from Browserify to something else, we don't know whether or not it will require that we wrap our own modules in IIFEs, so we're potentially just creating a very tedious job for someone to have to later go  back in and wrap everything back up.  

They're also just safer to use, in general, so if a framework isn't explictly saying don't wrap modules in IIFEs, I don't see why we wouldn't..

**Comment by jnmarcus** [comment]
this closes #89

**Pull request merged**